File: C++/Hard/WrongAnswer/recover-the-original-array_1.cpp
class Solution {
public:
    vector<int> recoverArray(vector<int>& nums) {
        
        int n = nums.size();
        sort(nums.begin(), nums.end());
        int k = (nums[n / 2] - nums[n / 2 - 1]) / 2;
        vector<int> ans;
        for (int i = 0; i < n; i += 2) {
            ans.push_back(nums[i] + k);
        }
        return ans;
    }
};

File: Java/Hard/WrongAnswer/recover-the-original-array_1.java
class Solution {
    public int[] recoverArray(int[] nums) {
        
        Arrays.sort(nums);
        int n = nums.length / 2;
        int[] res = new int[n];
        for (int i = 0; i < n; i++) {
            res[i] = (nums[i] + nums[n + i]) / 2;
        }
        return res;
    }
}